src/main/java/by/onliner/page/AboutCompanyPage.java

```diff
@@ -0,0 +1,44 @@
+package by.onliner.page;
+
+import by.onliner.core.elements.widget.Text;
+import by.onliner.core.utils.Waiter;
+import io.qameta.allure.Step;
+import org.openqa.selenium.support.FindBy;
+
+public class AboutCompanyPage extends AbstractPage{
+
+    @FindBy(xpath = "//div[@class='news-header__title']/h1")
+    private Text pageMessage;
+
+    public static final String ABOUT_LINK = "/about";
```

Comment:
Why is it public?

It's a good practice to define usual variables first and define web elements after them. Could you please rearrange variables?

Reply:
Fixed